src/micro_http/src/server.rs

```diff
@@ -198,13 +198,13 @@ impl<T: Read + Write> ClientConnection<T> {
 /// HTTP Server implementation using Unix Domain Sockets and `EPOLL` to
 /// handle multiple connections on the same thread.
 ///
-/// The function that does the data exchange is `handle_notifications`.
+/// The function that does the data exchange is `requests`.
```

Comment:
I can't say for sure what this line refers to. `data exchange` is a vague description of what happens when we call `requests`, or even after the call. As far as the implementation goes, `requests` simply parses the HTTP requests, and returns them, but as a bonus, handles new connections and sends the responses as well for parsed requests. This is a mixed functionality provided by a single method, with a name which can not encompass all this functionality. I think that for now it will be best to address the lack of matching between the docs and the actual implementation, and get back to this issue in another PR. 

```suggestion
/// The function that handles incoming connections, parses incoming requests and sends responses for awaiting requests is `requests`.
```

Reply:
done 👍